packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -832,12 +832,25 @@ const ReleaseDetailsPage = () => {
     );
   }
 
-  const title = (isSuccessDetails && data?.data?.name) || '';
+  const releaseData = (isSuccessDetails && data?.data) || null;
+
+  const title = releaseData?.name || '';
+  const scheduledAt = releaseData?.scheduledAt ? new Date(releaseData.scheduledAt) : null;
+  const timezone = releaseData?.timezone;
+
+  let time = '';
+  if (scheduledAt) {
+    const hours = scheduledAt.getHours();
+    const minutes = scheduledAt.getMinutes();
+    time = `${hours.toString().padStart(2, '0')}:${minutes.toString().padStart(2, '0')}`;
+  }
```

Comment:
I think you should be using something like:

https://github.com/strapi/design-system/blob/c4ecb525c730cd100c6a7a1fe03d7dabea759e9b/packages/strapi-design-system/src/hooks/useDateFormatter.ts#L32

to handle this sort of stuff, string manipulation is very messy and prone to errors.